src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -1,17 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <AssemblyVersion>4.0.3.0</AssemblyVersion>
-    <!-- Bump the below version each time we need to service a new version and ensure GeneratePackageOnBuild is set to true -->
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <ServicingVersion>2</ServicingVersion>
-
-    <!-- Package version properties -->
-    <VersionPrefix>4.5.$(ServicingVersion)</VersionPrefix>
-    <PackageValidationBaselineVersion>4.5.$([MSBuild]::Subtract($(ServicingVersion), 1))</PackageValidationBaselineVersion>
```

Comment:
These lines made more sense to me, as they were saying that when `<GeneratePackageOnBuild>` is true (now `<IsPackable>`), then we will be building the _next_ version that we would push to nuget.

In nuget.org, the latest published version of System.Buffers is 4.5.1: https://www.nuget.org/packages/System.Buffers/4.5.1

which means that with the code modified in this PR, we would have a collision.

Plus, the `<ServicingVersion>` matches what we use in servicing in runtime.

Reply:
How would this logic work when updating to a higher minor / major? I'm updating to 4.6.0 because the TFM breaking changes require at least a minor version upgrade.

Reply:
Oh I misread that 4.6.0 for 4.5.0. Got it.